frontend/src/views/files/Listing.vue

```diff
@@ -385,7 +385,7 @@ export default {
     req: function () {
       if (window.sessionStorage.getItem("listFrozen") !=="true" && window.sessionStorage.getItem("modified") !=="true"){
         // Reset the show value
-        this.showLimit = 50;
+        this.showLimit = this.req.numDirs + this.req.numFiles;
```

Comment:
showLimit here will not affect the real showLimit when the page mounts. But this showLimit here will reset the scrollHeight, and allow the browser to scoll down enough to get to the position we need